.github/workflows/tests.yml

```diff
@@ -94,12 +94,27 @@ jobs:
     uses: ./.github/workflows/tests-frontend.yml
     secrets: inherit
 
+  frontend-tests-coverage:
+    needs: frontend-tests
+    uses: ./.github/workflows/code-coverage.yml
+    secrets: inherit
+    with:
+      resource_name: report-fe
+      type: unit
```

Comment:
```suggestion
```

since the `unit` is the default value.

Or add `type: unit` on the backend-test-coverage.